### ISSUE TITLE
vcf to maf conversion

### ISSUE THREAD

**Issue by kunstner** [nf-core/sarek#785]
### Description of feature

Hi,

I was wondering whether it is possible to integrate vcf2maf as well into sarek to directly convert annotated vcf files to maf.

https://github.com/mskcc/vcf2maf

Best,
 Axel

**Comment by FriederikeHanssen** [comment]
Yes we have talked about it quite a bit. However, before we do that we need to find a solution for snpeff annotated files and the merged ones. vcf2maf only works for VEP.

**Comment by kunstner** [comment]
Many thanks for the quick reply. Right, the conversion is VEP only. I've forgotten that issue.
